Source/Scene/TextureAtlas.js

```diff
@@ -201,16 +223,25 @@ define([
                 pixelFormat : textureAtlas._pixelFormat
             });
 
-            // Copy old texture into new using an fbo.
             var framebuffer = new Framebuffer({
                 context : context,
-                colorTextures : [textureAtlas._texture]
+                colorTextures : [newTexture],
+                destroyAttachments : false
+            });
+
+            var command = textureAtlas._copyCommand;
+            command.renderState = RenderState.fromCache({
+                viewport : new BoundingRectangle(0, 0, oldAtlasWidth, oldAtlasHeight)
             });
+
             framebuffer._bind();
-            newTexture.copyFromFramebuffer(0, 0, 0, 0, oldAtlasWidth, oldAtlasHeight);
+            command.execute(textureAtlas._context);
```

Comment:
`TextureAtlas` is some of the oldest code in Cesium - it predates commands by at least a year.

I am not sure that `TextureAtlas` should keep a reference to `context`.  No one else does that.  This allows `TextureAtlas` to make WebGL calls whenever it wants regardless of the global state.  This is not good for scheduling `requestAnimFrame` and for interop for other WebGL engines (#648).  I wonder if it is even the source of this issue.

Perhaps we could replace `_context` with an `update` function and use `ComputeCommand`s?  How hard is that refactor?  Is it too architect astronaut-ish?